how-to/customize-workspace/client/src/launch.ts

```diff
@@ -254,6 +255,11 @@ export async function launch(appEntry: App) {
 					`App with id: ${appEntry.appId} could not be launched as it is of manifestType: ${appEntry.manifestType} and the endpoint: ${appEntry.manifest} is not available.`
 				);
 			}
+		} else if (appEntry.manifestType === "connection") {
```

Comment:
Perhaps we should have manifest type const definitions somewhere as well

Reply:
We have a json file but that is in dev extensions.